src/posthog-featureflags.ts

```diff
@@ -27,11 +32,18 @@ export const parseFeatureFlagDecideResponse = (response: Partial<DecideResponse>
                 })
         } else {
             // using the v2+ api
+            let newFeatureFlags = flags
```

Comment:
flyby that we also need to cater to the case when decide returns non-200 response, and doesn't obliterate flags, but keeps old flags instead.

Reply:
I think this handling is missing [here](https://github.com/PostHog/posthog-js-lite/blob/master/posthog-core/src/index.ts#L437)?

Reply:
the res.featureFlags wouldn't exist on non-200 calls

Reply:
Oh, sigh the js implementations could really use alignment

Reply:
> Oh, sigh the js implementations could really use alignment

I think everyone who has worked with them agrees with this statement.